app/dao/service_whitelist_dao.py

```diff
@@ -13,5 +14,5 @@ def dao_add_and_commit_whitelisted_contacts(objs):
 
 
 def dao_remove_service_whitelist(service_id):
-    return ServiceWhitelist.query.filter(
-        ServiceWhitelist.service_id == service_id).delete()
+    stmt = delete(ServiceWhitelist).where(ServiceWhitelist.service_id == service_id)
+    db.session.execute(stmt)
```

Comment:
.commit()?

Reply:
![image](https://github.com/department-of-veterans-affairs/notification-api/assets/16893311/2b03639e-dfae-40e8-ae87-bdc730f0f5dd)